src/pages/_app.tsx

```diff
@@ -30,10 +38,12 @@ export default function MyApp({ Component, pageProps }: AppProps<InitialPropInte
       `}</style>
       <SEO />
       <ThemeProvider>
-        <ToastContainer />
-        <PageLayout user={pageProps?.user}>
-          <Component {...pageProps} />
-        </PageLayout>
+        <MUIThemeProvider theme={theme}>
+          <ToastContainer />
+          <PageLayout user={pageProps?.user}>
+            <Component {...pageProps} />
+          </PageLayout>
+        </MUIThemeProvider>
```

Comment:
question: have we been using MUI for portal v2, or does this PR introduce its use?